client/src/plus/deep-dives/overview/index.scss

```diff
@@ -206,13 +177,23 @@
     &::before {
       background: transparent url("./assets/planning-for-browser-support.svg")
         center center no-repeat;
+      background-size: cover;
+
+      @media #{layout.$mq-small-desktop-and-up} {
+        background-size: 100%;
+      }
     }
   }
 
   .your-browser-support-toolkit {
     &::before {
       background: transparent url("./assets/your-browser-support-toolkit.svg")
         center center no-repeat;
+      background-size: cover;
+
+      @media #{layout.$mq-small-desktop-and-up} {
+        background-size: 100%;
```

Comment:
I want the preview images to be squares for desktop, so `background-size: 100%;` isn't necessary. But do add `    width: 100px;` and get rid of the `flex: 0 0 30%;` from `.overview-content-container .series-item:before`